packages/website/docs/_components_pages/fiori/ShellBar/ShellBar.mdx

```diff
@@ -1,5 +1,6 @@
 import Basic from "../../../_samples/fiori/ShellBar/Basic/Basic.md";
 import Advanced from "../../../_samples/fiori/ShellBar/Advanced/Advanced.md";
+import WithAdditionalContext from "../../../_samples/fiori/ShellBar/WithAdditionalContext/WithAdditionalContext.md";
```

Comment:
WithAdditionalContext sounds too technical,
Currently we have the BasicSample and Profile Area, so maybe add Content Area
or rename Profile Area to With Profile Area and add With Content Area
or something among these lines